code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -87,6 +140,13 @@
 		if(loaded)
 			to_chat(user, "<span class='notice'>You fill [src] with the monkey cubes stored in [O]. [src] now has [monkeys] monkey cubes stored.</span>")
 		return
+	else if (istype(O, /obj/item/multitool))
```

Comment:
use the helper

Reply:
this this `/obj/machinery/computer/camera_advanced/xenobio/multitool_act(mob/living/user, obj/item/multitool/I)` the helper?